### ISSUE TITLE
[Question] How to define a number in JSON without initial value?

### ISSUE THREAD

**Issue by jwillmer** [Starcounter/Starcounter#3144]
This works:
```html 
<input type="number" placeholder="type some number" value="{{model.Number$::input}}">
```

```json
"Number$": 0,
```

But I like to set the initial value to undefined to display the input placeholder:
```json
"Number$": null,
```

If I use null the mapping to the backend does not work :(

**Comment by per-samuelsson** [comment]
Guess we could consider supporting all the [Nullable types](https://msdn.microsoft.com/en-us/library/1t3y8s4s.aspx) and some syntax that maps to that, like "Number" : "?". 

AFAIK, its not supported as of today, so maybe we can find some nice workaround next week?

@chrhol, was this ever discussed?

**Comment by chrhol** [comment]
> @chrhol, was this ever discussed?

No, not that I can remember. It sounds like a good feature enhancement to support nullable types though.

**Comment by miyconst** [comment]
@jwillmer was your question answered? Can we close this issue?

**Comment by jwillmer** [comment]
As I understand it is not possible at the moment so the question is answered. I did not close it because I was not sure if you like to support it and use the question as a reminder to implement the feature. If you do not implement it then close it.

**Comment by miyconst** [comment]
We do plan to support, but we already have set of issues for supporting null-able numbers, that is why this one can be closed.
